### PR TITLE
feat: add reproducible pipelines

### PR DESCRIPTION
**Pull request by frezbo**
This helps automate the reproducibility test.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,14 +42,19 @@ target-%: ## Builds the specified target defined in the Pkgfile. The build resul
...
+
+reproducibility-test-local-%: ## Builds the specified target defined in the Pkgfile using the local output type. The build result will be output to the specified local destination.
+	@rm -rf _out1/ _out2/
+	@$(MAKE) local-$* DEST=_out1
+	@$(MAKE) local-$* DEST=_out2 TARGET_ARGS="--no-cache"
+	@touch -ch -t $$(date -d @$(SOURCE_DATE_EPOCH) +%Y%m%d0000) _out1 _out2
+	@diffoscope _out1 _out2
+	@rm -rf _out1/ _out2/
+
+foo:
```
Comment (smira) [inline Makefile:56]: debug?

**Review by smira — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (frezbo) [reply Makefile]: was trying to parse the git log time format to unix timestamp

**Pull request merged**